File: Q19/main.py
n= int(input())

arr=[]

for i in range(n):
    arr.append(int(input()))

max=secmax=arr[0]

for i in range(n):
    if arr[i]>max:
        secmax = max
        max = arr[i]
    elif arr[i]>secmax and max!=arr[i]:
        secmax = arr[i]
    
print(secmax)

File: Q19/main.cpp
#include<iostream>
using namespace std;

int main(){
    int n;
    cin>>n;
    int arr[n];
    for(int i=0;i<n;i++){
        cin>>arr[i];
    }
    int max=arr[0],secmax=arr[0];
    for(int i=0;i<n;i++){
        if(arr[i]>max){
            secmax = max;
            max=arr[i];
        }
        else if(arr[i]>secmax && arr[i]!=max)
            secmax=arr[i];
    }
    cout<<secmax;
    return 0;
}